internal/readers/file.go

```diff
@@ -0,0 +1,20 @@
+package readers
+
+import "os"
+
+type FileReader struct {
```

Comment:
question: if this is used only in tests, do we need it to be exposed, even under the internal package?